bin/cron_script.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# GENERAL IDEA for this cron script: we want to make sure that every time security updates are issued,
+# then building of a new kubenow image release is triggered.
+# This script will perform the necessary modifications in order to use the latest stable kubenow image
+# as source image for our packer builders and run security updates
+
+# Exit immediately if a command exits with a non-zero status
+set -e
+
+# Setting attribute source image name to the right value for each cloud provider
+if [ "$HOST_CLOUD" = 'aws' ]; then
+  echo -e "Running AWS Packer builder...\n"
+
+  # Installing aws-cli
+  pip install awscli --upgrade
+
+  # Extracting ami ID of latest kubenow stable (usually in the format of "ImageId:" "ami-xxxxxxx",)
+  kubenow_latest_amiId=$(aws ec2 describe-images --filters "Name=name,Values=kubenow-v$SUPPORTED_STABLE_TAG" "Name=owner-id,Values=105135433346" |
+    grep "ImageId" |
+    awk '{ print $2 }' |
+    sed -e 's/^"//' -e 's/",$//')
+
+  # Check whether or not id string is empty
+  if [ -n "$kubenow_latest_amiId" ]; then
+    # Updating source image id that will be used for AWS packer builder
+    echo -e "Kubenow latest stable ami ID is: $kubenow_latest_amiId .\n"
+    export AWS_SOURCE_IMAGE_ID="$kubenow_latest_amiId"
+    echo -e "AWS Source Image ID is: $AWS_SOURCE_IMAGE_ID\n"
+
```

Comment:
Please remove empty space to have same style as the other scripts in the project (if that's the style 🙂 )

Reply:
Sorry, not sure which empty space you are referring to. Could you be more specific? Just want to avoid making you repeat the same comment by performing a misinterpreted change :-)

Reply:
Alright, got it!